tests/network/localnet/liblocalnet.py

```
@@ -1,15 +1,19 @@
 import contextlib
 import logging
 import uuid
-from typing import Final, Generator
+from typing import Generator
```

Ruff (0.15.9)

[warning] Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)